### DISCUSSION TITLE
Help Getting Started with rtsp-simple-server

### DISCUSSION THREAD

**Discussion by seabass1217** [bluenviron/mediamtx#607] [Questions]
Hi,
New to video streaming...
My development development HW is a jetson xavier AGX and I have downloaded and extracted rtsp-simple-server, but when I try to start the sever with 'sudo ./rtsp-simple-server' command I get the below  error.  I'm not sure what I did wrong.  Any help is appreciated.

![Screenshot from 2021-10-01 07-58-24](https://user-images.githubusercontent.com/86954802/135618237-a7416a23-1b4a-4735-a6d6-ef36fc4d93f8.png)

**Comment by aler9** [comment]
Make sure you downloaded the "arm64" binary of the server, that's the one compatible with the Xavier AGX.

[a comment by an automated account is not shown]
